doc/config.md

```diff
@@ -224,6 +224,12 @@ Only one tag used as filter for index field Tag1, see graphite_tagged table [str
   # offset beetween now and until for select short cache timeout
   short-offset = 0
 
+[feature-flags]
+ # if true, prefers carbon's behaviour on how tags are treated
```

Comment:
Add more definition for this options in deploy/doc/config.md (with some exaples). May be some table with queries.

Reply:
Added in [d569edc](https://github.com/go-graphite/graphite-clickhouse/pull/266/commits/d569edc96169fc4fecb1cc96510f1256da271b81)